### PR TITLE
feat(dockerfile): add new 3.3 base images

### PR DESCRIPTION
**Pull request by dholmes**
Adds the set of ruby-3.3 images: https://github.com/articulate/docker-ruby/issues/73

### CODE REVIEW

File: 3.3-bullseye-slim-minimal/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# tags=articulate/articulate-ruby:3.3-bullseye-slim-minimal
```
Comment (mloberg) [inline 3.3-bullseye-slim-minimal/Dockerfile:1]: This is our old pattern, follow the 3.2 structure

**Review by mloberg — CHANGES_REQUESTED** [review]

File: 3.3-bullseye-slim-minimal/Dockerfile (hunk 1 above)
Comment (mloberg) [reply 3.3-bullseye-slim-minimal/Dockerfile:1]: Oh I didn't scroll down far enough. We don't need to maintain any articulate-ruby images anymore. Just articulate/ruby

Comment (dholmes) [reply 3.3-bullseye-slim-minimal/Dockerfile:1]: Oh, gotcha!  That's what you were getting at in the ticket..just the 3.3/base and 3.3/node images then. :+1:

Comment (dholmes) [reply 3.3-bullseye-slim-minimal/Dockerfile]: Go ahead and resolve if this is now corrected. (want to make sure I understood)

**Review by mloberg — APPROVED** [review]
I'll create a ticket for this, but we can remove the 2.7 build since that's been EOL'd for 9 months now

**Pull request merged**